File: R/tests/testthat/test_omsfree_5bias.R
context("Test omsfree_5bias")
library(hBayesDM)

test_that("Test omsfree_5bias", {
  # Do not run this test on CRAN
  skip_on_cran()

  expect_output(omsfree_5bias(
      data = "example", niter = 10, nwarmup = 5, nchain = 1, ncore = 1))
})

File: Python/tests/test_omsfree_5bias.py
import pytest

from hbayesdm.models import omsfree_5bias


def test_omsfree_5bias():
    _ = omsfree_5bias(
        data="example", niter=10, nwarmup=5, nchain=1, ncore=1)


if __name__ == '__main__':
    pytest.main()
